### DISCUSSION TITLE
Add confirmation prompt when deleting an account

### DISCUSSION THREAD

**Discussion by ghost** [FireCubeStudios/Protecc#65] [Ideas]
Currently when we click on the trash can icon on any account in the app, the account gets deleted. it can happen accidentally, there better be a prompt for confirmation before proceeding with deletion.

**Comment by FireCubeStudios** [comment]
Yeah the next release will actually require entering editing mode to delete with a confirmation which should prevent accidents

**Reply by ghost** [reply]
That's good to know, thank you
